src/main/java/edu/harvard/iq/dataverse/search/SolrSearchResult.java

```diff
@@ -1259,18 +1265,25 @@ public void setDatasetValid(Boolean datasetValid) {
 	}
 
 	public boolean isValid(Predicate<SolrSearchResult> canUpdateDataset) {
-        if (this.datasetValid) {
-            return true;
-        }
-        if (!this.getType().equals("datasets")) {
-            return true;
-        }
-        if (this.isDraftState()) {
-            return false;
-        }
-        if (!JvmSettings.UI_SHOW_VALIDITY_LABEL_WHEN_PUBLISHED.lookupOptional(Boolean.class).orElse(true)) {
-            return true;
-        }
+		if (this.datasetValid) {
+			return true;
+		}
+		if (!this.getType().equals("datasets")) {
+			return true;
+		}
+		if (this.isDraftState()) {
+			return false;
+		}
+		if (!JvmSettings.UI_SHOW_VALIDITY_LABEL_WHEN_PUBLISHED.lookupOptional(Boolean.class).orElse(true)) {
+			return true;
+		}
```

Comment:
Are these tabs? Can they please be reverted to spaces?

Reply:
My IntelliJ IDEA says it is 4 spaces tab spacing. Anyway, I fixed this because indentation was broken `L1262`.